go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/AlecAivazis/survey/v2 v2.2.14
 	github.com/Masterminds/semver/v3 v3.1.1
-	github.com/briandowns/spinner v1.16.0
+	github.com/briandowns/spinner v1.15.0
```

Comment:
Was downgrading this dependency intentional?

Reply:
Correct @sanderblue we found a regression in the spinner latest (1.16) version, it's duplicating the message when stopping somehow. We reverted to fix the issue.
cc @ctrombley